example/config.yml

```diff
@@ -8,6 +8,16 @@ cumulus-sit:
   apiUsername: jasmine
   pdrNodeNameProviderBucket: cumulus-sit-pdr-node-name-provider
 
+cumulus-es:
```

Comment:
These buckets should probably be removed before merging, right?

Reply:
cumulus-lp is used for sit testing.  cumulus-es isn't actually used, but we might use it if we need to test es performance.